Gemfile.lock

```diff
@@ -429,6 +431,7 @@ GEM
     zeitwerk (2.6.6)
 
 PLATFORMS
+  arm64-darwin-23
```

Comment:
Is this because you're using an apple laptop and ARM processor?

Reply:
Yes - and i could either solve it by forcing nokogiri to use the ruby version `bundle config force_ruby_platform true` or adding my platform as recommended here: https://nokogiri.org/tutorials/installing_nokogiri.html#supported-platforms.